LICENSES/LicenseRef-MSLA.txt

```diff
@@ -0,0 +1,842 @@
+
+   7.2.2. Silicon Labs hereby grants to Licensee a perpetual, worldwide,
+   non-exclusive, no-charge, royalty-free, irrevocable (except as stated
+   in this section) patent license to make, have made, use, offer to sell,
+   sell, import, and otherwise transfer the Silicon Labs Open Source Code,
+   where such license applies only to those patent claims licensable by
+   Silicon Labs that are necessarily infringed by Licensee’s use of the
+   Silicon Labs Open Source Code or by combination with any other device
+   or software.
+
+   7.2.3. Licensee may add Licensee’s own copyright statement to
+   Licensee’s modifications of Silicon Labs Open Source Software and may
+   provide additional or different license terms and conditions for use,
+   reproduction, or distribution of such modifications, or for any such
+   Derivative Works as a whole, provided Licensee’s use, reproduction, and
+   distribution of the Silicon Labs Open Source Software otherwise
+   complies with the conditions stated in this License.
+
+   7.2.4. Licensee may reproduce and distribute copies of the Silicon Labs
+   Open Source Code or Derivative Works thereof in any medium, with or
+   without modifications, and in Source Code or Object Code form, provided
+   that Licensee meets the following conditions: (a) Licensee must give
+   any other recipients of the Silicon Labs Open Source Code or Derivative
+   Works a copy of this License; and (b) Licensee must cause any modified
+   files to carry prominent notices stating that Licensee changed the
+   files; and (c) Licensee must retain, in the Source Code form of any
+   Derivative Works that Licensee distributes, all copyright, patent,
+   trademark, and attribution notices from the Source Code form of the
+   Silicon Labs Open Source Code, excluding those notices that do not
+   pertain to any part of the Derivative Works; and (d) If the Silicon
+   Labs Open Source Code includes a "NOTICE" text file as part of its
+   distribution, then any Derivative Works that Licensee distributes must
+   include a readable copy of the attribution notices contained within
+   such NOTICE file, excluding those notices that do not pertain to any
+   part of the Derivative Works, in at least one of the following places:
+   within a NOTICE text file distributed as part of the Derivative Works;
+   within the Source Code form or documentation, if provided along with
+   the Derivative Works; or, within a display generated by the Derivative
+   Works, if and wherever such third-party notices normally appear. The
+   contents of the NOTICE file are for informational purposes only and do
+   not modify the License. Licensee may add Licensee’s own attribution
+   notices within Derivative Works that Licensee distributes, alongside or
+   as an addendum to the NOTICE text from the Silicon Labs Open Source
+   Code, provided that such additional attribution notices cannot be
+   construed as modifying the License.
```

Comment:
It seems `lynx` does not do a perfect job. I suggest to reformat this section to align sub-items (my comment could also apply in various places with sub-items).

Reply:
If I had add to author the file I would do that as well but here I prefer something deterministic this way we can spot changes (if any occurs)

Reply:
Make sense.